packages/venice-types/src/IAlert.ts

```diff
@@ -0,0 +1,23 @@
+enum type {
+  'success',
+  'danger',
+  'toast',
+  'vip',
```

Comment:
`vip` describe a business rule, maybe we can add a most generic name here.

Reply:
You hava sugestions to rename this?

Reply:
maybe... `star`?

Reply:
featured?

Reply:
Nice! `feature` sounds good 😁